config/settings/base.py

```
@@ -24,11 +24,17 @@
-# SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = os.getenv('SECRET_KEY', 'django-insecure-q9my7am$yvak#jo6)gh&o%gx-0*p2u6!-lfy-72h+=)-(jxq8t')
+# Parse DEBUG from environment with safe defaults
+DEBUG = os.getenv('DEBUG', 'False').lower() in ('true', '1', 't', 'yes', 'y')
 
-# SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = os.getenv('DEBUG', 'False').lower() == 'true'
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = os.getenv('SECRET_KEY')
+if not SECRET_KEY and DEBUG:
+    import warnings
+    warnings.warn('SECRET_KEY not set. Using a development default. Set SECRET_KEY in .env for production.')
```

Ruff (0.12.2)

No explicit `stacklevel` keyword argument found

Set `stacklevel=2`

(B028)